packages/package_info_plus/package_info_plus_web/test/package_info_plus_web_test.dart

```diff
@@ -16,6 +16,7 @@ void main() {
     'build_number': '1',
     'package_name': 'io.flutter.plugins.packageinfoexample',
     'version': '1.0',
+    'build_signature': '',
```

Comment:
Something with a value would be better to test the function.

```suggestion
    'build_signature': 'test_signature',
```

Reply:
yes, but the code does not set this variable.

Reply:
Ok, just found it, nevermind.
`Contains the signing key signature (hex) on Android and empty string on everything else.`
Do we need to 'test' this?

Reply:
since it should be empty in case of web, i don't see a downside in testing it. bonus to prevent the mistake like i made of not reverting.

<details>
<summary>what and why did i do that</summary>

well, i wanted to verify what the different functions return in case of chrome extensions, but i can't run the via flutter run as it's that. so i thought of using one of the PackageInfo model fields to get the url. but after testing, i forgot to remove it.
<details>